src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py

```diff
@@ -1786,10 +1785,14 @@ def run(self):
       # If we did not pick a fuzz target to fuzz with the engine, then return
       # early to save the fuzz targets that are in the build for the next job to
       # pick.
-      output_to_report_targets = self.record_fuzz_targets_to_output(
-          build_setup_result)
+      self.fuzz_task_output.fuzz_targets.extend(build_setup_result.fuzz_targets)
+      if not self.fuzz_task_output.fuzz_targets:
+        logs.log_error('No fuzz targets.')
+
       if not fuzz_target:
-        return output_to_report_targets
+        return uworker_msg_pb2.Output(
+            fuzz_task_output=self.fuzz_task_output,
```

Comment:
You set the fuzz_task_output here but not in the consequent Output's returned in case of error. 
Is that intentional?

Reply:
Yep.